pkgs/development/python-modules/mirakuru/default.nix

```diff
@@ -13,18 +14,23 @@
 
 buildPythonPackage rec {
   pname = "mirakuru";
-  version = "2.5.2";
+  version = "2.5.3";
   format = "pyproject";
 
-  disabled = pythonOlder "3.8";
+  disabled = pythonOlder "3.9";
 
   src = fetchFromGitHub {
     owner = "ClearcodeHQ";
     repo = "mirakuru";
     rev = "refs/tags/v${version}";
-    hash = "sha256-I1TKP0ESuBMTcReZf0tryjvGpSpwzofwmOiQqhyr6Zg=";
+    hash = "sha256-blk4Oclb3+Cj3RH7BhzacfoPFDBIP/zgv4Ct7fawGnQ=";
   };
 
+  patches = [
+    # https://github.com/ClearcodeHQ/mirakuru/pull/810
+    ./tmpdir.patch
```

Comment:
How about using this URL instead? https://github.com/ClearcodeHQ/mirakuru/commit/68526a308fb6cbecba6896ea5ee295821174f9a5.patch

Reply:
Fair enough, I won't block it then.